### PR TITLE
IW-2179 | Global Footer update

### PR DESCRIPTION
**Pull request by ustczanka**
https://wikia-inc.atlassian.net/browse/IW-2179

### CODE REVIEW

File: extensions/wikia/DesignSystem/services/DesignSystemGlobalFooterService.class.php [hunk 1]
```diff
@@ -15,10 +15,13 @@ public function section() {
 		$this->response->setValues( [
 			'model' => $this->getVal( 'model' ),
 			'name' => $this->getVal( 'name' ),
-			'parentName' => $this->getVal( 'parentName' )
 		] );
 	}
 
+	public function image() {
+		$this->setVal( 'model', $this->getVal( 'model' ) );
+	}
```
Comment (mszabo-wikia) [inline extensions/wikia/DesignSystem/services/DesignSystemGlobalFooterService.class.php]: there's no need for a method here, you can render the template directly with renderPartial()

File: includes/wikia/models/DesignSystemGlobalFooterModel.class.php [hunk 2]
```diff
@@ -199,17 +182,114 @@ public function getData() {
...
+					'name' => 'wds-company-store-logo-ddb',
+					'caption' => [
+						'type' => 'text',
+						'value' => 'D&D Beyond'
+					],
+				],
+				'links' => [
+					[
+						'type' => 'link-image',
+						// 'image' is deprecated, use 'image-data' instead
```
Comment (mszabo-wikia) [inline includes/wikia/models/DesignSystemGlobalFooterModel.class.php]: does anyone still use it?

Comment (ustczanka) [reply includes/wikia/models/DesignSystemGlobalFooterModel.class.php]: I'm not really sure, so I'll leave it for now. We'll throw it anyway sometime soon.

Comment (mszabo-wikia) [reply includes/wikia/models/DesignSystemGlobalFooterModel.class.php]: soon ™️ :))

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**